.changeset/spicy-moons-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-insights-backend-module-jsonfc': minor
```

Comment:
I think this should still be a patch unfortunately (since we are pre 1.x.x). Additional functionality and no breakages since those new args are optional.

Reply:
Done!